LoggingKata/TacoParser.cs

```diff
@@ -9,10 +9,32 @@ public class TacoParser
         
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+            //logger.LogInfo("Begin parsing");
+            TacoBell tacoBell = new TacoBell();
+            Point point = new Point();
+            var cells = line.Split(',');
+            if (cells.Length < 3)
+            {
+                return null;
+            }
+            string name = cells[2];
+            double longitude = double.Parse(cells[1]);
+            double latitude = double.Parse(cells[0]);
```

Comment:
These lines are risky if cells[0] or cell[1] can't be parsed as a double. Could a TryParse be used here instead?